css/properties.json

```diff
@@ -3240,6 +3240,26 @@
     ],
     "status": "standard"
   },
+  "font-variation-settings": {
+    "syntax": "normal | [ &lt;string&gt; &lt;number&gt; ] #",
```

Comment:
Nit: We're not adding spaces between `]` and the multiplier. Can you please remove it?

Sebastian